lib/cli.rb

```diff
@@ -761,13 +753,20 @@ def self.parse_inspect_command_options(host, options)
     c.action do |global_options,options,args|
       Machinery::Ui.use_pager = options["pager"]
 
+      if options[:html]
+        begin
+          check_port_validity(@config.http_server_port)
+        rescue Machinery::Errors::ServerPortError => e
+          raise Machinery::Errors::InvalidCommandLine.new(e.message +  " The port can be " \
```

Comment:
Unnecessary spacing detected.<br>Operator `+` should be surrounded with a single space.